docs/submission/clinical-data-validation-rules.md

```diff
@@ -37,19 +37,45 @@ These identifier fields allow for linking across the different clinical events a
 
 ### Time Intervals
 
-To prevent potential identification of donors, actual calendar dates are not permitted. The timing of different clinical events are collected in days counted from the date of primary diagnosis.
+To prevent potential identification of donors, actual calendar dates are not permitted. The timing of different clinical events are collected in days counted from the date of primary diagnosis. The date of primary diagnosis is the date on which a definitive diagnostic procedure was performed. Validation checks are in place to ensure the values submitted for the different time interval fields make sense according to the following assumptions:
+
+- The `age_at_primary_diagnosis` is used as the reference time point.
+- The day the patient dies is the clinical endpoint (`survival_time`).
+
+Examples of time interval validation checks:
+
+- If a patient's `vital_status` is `Deceased`, all time intervals must be less than or equal to `survival_time`.
+- The `relapse_interval` must be less than the `interval_of_followup` in the Follow Up the relapse was recorded.
+- If a follow up is associated with a particular treatment (via the `submitter_treatment_id`), the `interval_of_followup` must be greater than the `treatment_start_interval`.
+
 
 ### Donors Older than 90 years old
 
 Since the occurrence of individuals over the age of 90 is rare, it is therefore considered a potentially identifiable value. Thus, the allowed value for the `age_at_diagnosis` field is capped at 90.
 
+## Cross Field Validations
 
+A number of cross-field consistency checks within files are implemented to ensure quality control and data correctness. This involves requiring the value of another field to validate the current field. The cross-field validation checks are implemented using Javascript. For the advanced user, you will be able to see the actual cross-field validation scripts in the Dictionary Viewer by clicking on `View Script`. Examples include:
```

Comment:
change "This involves requiring the value of another" to "This requires the value of another.."